src/map.h

```diff
@@ -40,8 +40,11 @@ class computer;
 struct itype;
 struct mapgendata;
 struct trap;
+struct oter_t;
+
 using trap_id = int_id<trap>;
-struct oter_id;
+using oter_id = int_id<oter_t>;
```

Comment:
These aren't nice names. I appreciate that wasn't invented here. How badly would line count be affected by more descriptive naming?